primitives/file-metadata/src/lib.rs

```diff
@@ -80,6 +81,20 @@ impl<const H_LENGTH: usize> Into<Hash<H_LENGTH>> for FileKey<H_LENGTH> {
     }
 }
 
+impl From<H256> for FileKey<32> {
+    fn from(hash: H256) -> Self {
+        let mut file_key = [0u8; 32];
+        file_key.copy_from_slice(hash.as_bytes());
+        Self(file_key)
+    }
+}
+
+impl Into<H256> for FileKey<32> {
+    fn into(self) -> H256 {
+        H256::from_slice(&self.0)
+    }
+}
```

Comment:
Not a huge fan of this, as it feels like we're braking the abstraction of this crate, but I agree on it's convenience, so 👍🏼